### ISSUE TITLE
EventCatalog sponsorship?

### ISSUE THREAD

**Issue by boyney123** [event-catalog/eventcatalog#509]
Hey folks,

I'm currently working on v2 of EventCatalog, super excited to share this with you all.

As EC continues to grow in our community, I'm curious what people think about sponsorship:

1. Does EC have what it takes to be sponsored?
2. Would your company ever consider sponsoring or paying for support?

Reasons I ask, is I'm trying to work out if there is a model where we can get full time work on EC.... 🚀

**Comment by LarsArtmann** [comment]
I think sponsors are always a great idea for OS project.
The way https://github.com/pnpm/pnpm handles them in the README.md and release notes is the best for all parties, IMO.

**Comment by zbigniew-malcherczyk-tg** [comment]
There are enterprises that Event Catalog helps and may be interested in supporting/sponsoring. Yet, some performance issues would need to be addressed. The current EC does not scale above 200 events/messages.

**Comment by boyney123** [comment]
> There are enterprises that Event Catalog helps and may be interested in supporting/sponsoring

Yeah something I'm considering, either sponsorship, open core model, or offer support/consulting on the side to make it work..... 

I would love to work on this more, just have to make it financially possible....

> The current EC does not scale above 200 events/messages

Interesting, I moved to a new file system reading method, updated NextJS and other stuff in v2, but I will make sure this get's addressed for sure. What kind of scale are you looking at?

> Did you consider switching from Next.JS to Go with [a-h/templ](https://github.com/a-h/templ) and HTMX?

Yeah might be a possible solution, I wanna test v2 out with the scale issue, but maybe going back to basics (Vite for example) could work here..... something to explore, but I noted this down.
